src/worker-helpers.js

```diff
@@ -119,8 +120,15 @@ export function getArgv(type, config, filePath, fileDir, givenConfigPath) {
   if (config.eslintRulesDir) {
     let rulesDir = resolveEnv(config.eslintRulesDir)
     if (!Path.isAbsolute(rulesDir)) {
+      const splitPath = rulesDir.split('/')
+      if (splitPath[0] === '~') {
```

Comment:
Not sure what you mean - my understanding is that `~` is shorthand for the users home directory. A path like /Users/~ will try to find a directory called ~, which is a legitimate use case and shouldn't be expanded.  But maybe I'm misunderstanding what you're saying?